plans/smlbench2/test/simple_add_get.go

```diff
@@ -0,0 +1,77 @@
+package test
```

Comment:
It seams that this file is just c&p from smlbench as the test for smlbench2 is on main.go

Reply:
Oh yeah, I was going to delete that file, I just forgot to. I was copying/pasting from it.